### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by csansoon**
Some small fixes:

- Fixed `api` Dockerfile to install all required modules correctly
- Fixed typo on `promptSnapshots` model
- Fixed casing on `users`, `sessions` and `acounts` models to follow our convention

### CODE REVIEW

File: apps/api/docker/Dockerfile [hunk 1]
```diff
@@ -9,15 +9,15 @@ FROM installer AS runner
...
+WORKDIR /app/apps/api
+
 RUN --mount=type=cache,id=pnpm,target=/pnpm/store \
     if [ "$NODE_ENV" = "production" ]; then \
         pnpm install --prod --frozen-lockfile; \
     else \
         pnpm install; \
     fi
 
-WORKDIR /app/apps/api
```
Comment (geclos) [inline apps/api/docker/Dockerfile:19]: why this change?

**Review by geclos — APPROVED** [review]
good catches on the table names 👌🏼

File: apps/api/docker/Dockerfile (hunk 1 above)
Comment (csansoon) [reply apps/api/docker/Dockerfile:19]: It wasn't installing the correct required packages, resulting in execution errors. Installing them after changing the workdir fixed it.
